lib/messages/subscription-list.js

```diff
@@ -1,10 +1,19 @@
 const { Message } = require('.');
 
 module.exports = class SubscriptionList extends Message {
-  constructor(repositories, accounts) {
+  constructor(subscriptions, repositories, accounts, showFeatureInformation = false) {
     super({});
+    this.subscriptions = subscriptions;
     this.repositories = repositories;
     this.accounts = accounts;
+    this.showFeatureInformation = showFeatureInformation;
+  }
+
+  getFormattedSubscriptionSettings(githubId, type) {
+    return this.subscriptions.find(subscription => (
+      subscription.type === type && subscription.githubId.toString() === githubId.toString()
+    ))
+      .getEnabledSettings().map(setting => `\`${setting}\``).join(', ');
```

Comment:
using the same logic used here: https://github.com/integrations/slack/blob/42e9489f348adcd6e156bbb81f3c683395b32def/lib/messages/flow/updated-settings.js#L10-L12